crates/holochain/src/core/validation.rs

```diff
@@ -10,11 +10,32 @@ use super::SysValidationError;
 use super::ValidationOutcome;
 
 /// Exit early with either an outcome or an error
+#[derive(Debug)]
 pub enum OutcomeOrError<T, E> {
```

Comment:
Should this just be a result, the `into_outcome` and `into_err` look a lot like the `ok` and `err` functions on a result

Reply:
Maybe so, but it looks like it was written this way in order to create some From impls, perhaps for convenience in the HDK. [This itself is used as the Err of another Result](https://github.com/holochain/holochain/blob/54401d8106f8e96e46fe402208158b9468c255e8/crates/holochain/src/core/workflow/app_validation_workflow/error.rs#L41). I have no idea what it's all hooked up to or what it might break in the HDK, and I don't want to touch any more than I already have.